kubernetes/kubernetes_service.go

```diff
@@ -84,60 +93,107 @@ func (in *IstioClient) GetDeployments(namespaceName string) (*v1beta1.Deployment
 	return in.k8s.AppsV1beta1().Deployments(namespaceName).List(emptyListOptions)
 }
 
+// GetDeployments returns a list of deployments for a given namespace.
```

Comment:
GetDeployments on GetService method.

Reply:
Sorry, my bad, that comment needs to be updated.